README.md

```diff
@@ -1,8 +1,14 @@
 # TensorFlow Cloud
 
-## What is this repo?
+The TensorFlow Cloud repository provides APIs that will allow to easily go from debugging, training, tuning your Keras and TensorFlow code in a local environment to distributed training/tuning in the cloud.
```

Comment:
(nit) 'in the cloud' sounds little bit funny. How about 'on Cloud' (capital C) or something?

Reply:
Updated!